src/Patterns/Path.php

```diff
@@ -69,24 +69,24 @@ class Path implements PathType
     private array $types = [];
 
     /**
-     * @var Variable
+     * @var ?Variable
      */
-    private Variable $variable;
+    private ?Variable $variable;
 
     /**
-     * @var int The minimum number of `relationship->node` hops away to search
+     * @var ?int The minimum number of `relationship->node` hops away to search
      */
-    private int $minHops;
+    private ?int $minHops;
 
     /**
-     * @var int The maximum number of `relationship->node` hops away to search
+     * @var ?int The maximum number of `relationship->node` hops away to search
```

Comment:
`?int` is not a valid PHPDoc type hint, it should be `int|null`. Same for the other PHPDoc comments in this PR (it's more work to add all these comments than it is to update them, so I won't put any more comments about it after this one :P)

Reply:
Resolved in 342cb91